File: tacas2020-benchmarks/vaphor/copy9.java
class vapexp{
	static int i, x, N;
	static int a0[], a1[], a2[], a3[], a4[];
	static int a5[], a6[], a7[], a8[], a9[];

static void main()
{
	N=Support.random();

	i=0;
        while(i<N)
	{
		a2[i] = a1[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a3[i] = a2[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a4[i] = a3[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a5[i] = a4[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a6[i] = a5[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a7[i] = a6[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a8[i] = a7[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a9[i] = a8[i];
		i = i + 1;
        }

	i=0;
        while(i<N)
        {
		a0[i] = a9[i];
		i = i + 1;
        }

	x=0;
        while(x<N)
        {
		assert(a1[x] == a0[x]);
		x = x + 1;
        }
}
}

File: tacas2020-benchmarks/booster/copy9.c
#define N

int main( ) {
	int a1[N];
	int a2[N];
	int a3[N];
	int a4[N];
	int a5[N];
	int a6[N];
	int a7[N];
	int a8[N];
	int a9[N];
	int a0[N];

	int a;
	int i;
	for ( i = 0 ; i < N ; i++ ) {
		a2[i] = a1[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a3[i] = a2[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a4[i] = a3[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a5[i] = a4[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a6[i] = a5[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a7[i] = a6[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a8[i] = a7[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a9[i] = a8[i];
	}
	for ( i = 0 ; i < N ; i++ ) {
		a0[i] = a9[i];
	}

	int x;
	for ( x = 0 ; x < N ; x++ ) {
		assert(  a1[x] == a0[x]  );
	}
	return 0;
}

